WalletWasabi/JsonConverters/MaxCoordinationFeeRateJsonConverterNg.cs

```diff
@@ -0,0 +1,33 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.JsonConverters;
+
+public class MaxCoordinationFeeRateJsonConverterNg : JsonConverter<Fraction>
+{
+	/// <inheritdoc />
+	public override Fraction Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+	{
+		// For backward compatibility, MaxCoordinationFeeRate used to be a percentage
+		// If it was migrated already then TokenType will be a string
+		if (reader.TokenType == JsonTokenType.Number)
```

Comment:
This uses the serialization format to take business decision, that's a hack. I strongly prefer some other mechanism.